samples/Calculator/src/Calculator.Console/Calculator.Console.csproj

```diff
@@ -0,0 +1,14 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <ItemGroup>
+    <ProjectReference Include="..\Calculator.Core\Calculator.Core.csproj" />
+  </ItemGroup>
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net7.0</TargetFramework>
```

Comment:
should we have common pros inside a directory prop/target to update in future to the newer framework easily?

Reply:
I will keep it like this. It's simple and many devs will not be familiar with prop/targets.

Reply:
I agree with Faisal - users should easily understand and in most cases they don't use props/targets. Update you can do with search/replace

Reply:
fine to me